elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -483,4 +509,33 @@ public RequestScope getRequestScope() {
             return requestScope;
         }
     }
+
+    private static void elideAcceptQuery(RequestScope requestScope, User opaqueUser,
+                                         Optional<MultivaluedMap<String, String>> queryParams, String path) {
+        if (requestScope.getElideSettings() != null) {
```

Comment:
When is getElideSEttings null?  Is this only for tests?  This really shouldn't happen.